code/modules/mob/living/living.dm

```diff
@@ -16,6 +16,8 @@
 
 	attack_icon = image("icon" = 'icons/effects/attacks.dmi',"icon_state" = "", "layer" = 0)
 
+	sight |= SEE_BLACKNESS
```

Comment:
Looks like this breaks thermo vision (blackness rendered over game objects like mobs), but requires further check.